src/components/Pages/Departments.js

```diff
@@ -12,18 +12,16 @@ import TileGroup from 'components/Tiles/TileGroup';
 
 const Departments = ({ departments, intl }) => {
   const departmentLinks = departments.map(d => ({
-    url: `/${intl.locale}/${d.slug}`,
```

Comment:
Does Tile Group's `export default injectIntl(TileGroup);` add the intl path to the url for us?